.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag (Optional, without 'v' e.g. 1.2.3)"
+
+jobs:
+  release:
+    name: Tag and release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bump version and push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          default_bump: false
+          custom_tag: ${{ github.event.inputs.tag }}
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v0.1.15
+        with:
+          name: ${{ steps.tag_version.outputs.new_tag }}
+          tag_name: ${{ steps.tag_version.outputs.new_tag }}
+          body: ${{ steps.tag_version.outputs.changelog }}
+          generate_release_notes: true
```

Comment:
You should also attach the relevant scripts/binaries to the release

Reply:
Right, you mean the new/changed files introduced in this release?

Reply:
I am a bit confused about this one. Which files do I attach?

Reply:
I think all, even the ones that haven't changed. If we look at this repo as a collection of utilities, on every release, all utilities should be attached to the GH release.

So for now, just your bash script.

Reply:
Added in https://github.com/canonical/rocks-toolbox/pull/2/commits/34f15ed72aa500aae3d6a1ce985f0ad1cb0d9f93.
Demo for the attached files: https://github.com/rebornplusplus/rocks-toolbox/releases/tag/v1.0.9.